app/serializers/article_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class ArticleSerializer < ActiveModel::Serializer
+  include Rails.application.routes.url_helpers
+  attributes :title, :link
+
+  def link
+    Rails.application.routes.url_helpers.api_v1_article_path(object)
```

Comment:
If you include `Rails.application.routes.url_helpers` on top, you don't need to use this again. Also, try `api_v1_article_url` instead.

Reply:
We tried using ``` api_v1_article_url``` but unsuccessfully 
``` 
Failure/Error: Rails.application.routes.url_helpers.api_v1_article_url(object)
     
     ArgumentError:
       Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true
``` 
Can't seams to figure out how to fix that error. So we just went with ```api_v1_path``` instead